app/services/domain/claim-decision.js

```diff
@@ -69,6 +69,23 @@ class ClaimDecision {
           .isLessThanMaximumDifferentApprovedAmount()
       }
     })
+    var check = false
+    this.claimExpenseResponses.forEach(function (expense) {
+      if (expense.status.includes(claimDecisionEnum.APPROVED)) {
+        check = true
+      } else if (expense.status.includes(claimDecisionEnum.APPROVED_DIFF_AMOUNT)) {
+        check = true
+      } else if (expense.status.includes(claimDecisionEnum.MANUALLY_PROCESSED)) {
+        check = true
+      } else if (expense.status.includes(claimDecisionEnum.REQUEST_INFORMATION)) {
+        check = true
+      }
+    })
```

Comment:
I tried that before and didnt work but it worked now. Maybe i did something wrong before.